src/master-playlist-controller.js

```diff
@@ -235,6 +237,8 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     this.syncController_ = new SyncController();
 
+    this.decrypter_ = work(Decrypter);
```

Comment:
Why do we create one decrypter here to share among loaders instead of creating a decrypter in each loader?